tests/artifactorydownload_test.go

```diff
@@ -34,7 +34,7 @@ func flatDownload(t *testing.T) {
 	if err != nil {
 		t.Error(err)
 	}
-	defer os.RemoveAll(workingDir)
+	defer assert.NoError(t, os.RemoveAll(workingDir))
```

Comment:
`RemoveAll` gets executed, we need to wrap it with func

Reply:
done